src/main/resources/assets/natura/lang/de_DE.lang

```diff
@@ -24,58 +24,58 @@ tile.natura.overworld_logs2.willow.name=Weidenholz
 tile.natura.overworld_logs2.willow.tooltip=Der Weinende
 tile.natura.overworld_logs2.eucalyptus.name=Eukalyptusholz
 tile.natura.overworld_logs2.eucalyptus.tooltip=Das rosa Holz
-tile.natura.overworld_logs2.sakura.name=Sakuraholz
+tile.natura.overworld_logs2.sakura.name=Blütenkirschenholz
```

Comment:
This should be "Blütenkirschholz".

Reply:
Umm my commit included that already. O.o :D

Reply:
Just saw the issue. sorry.
Corrected it.